controllers/storagecluster/volumesnapshotterclasses.go

```diff
@@ -94,7 +91,7 @@ func newSnapshotClassConfigurations(instance *ocsv1.StorageCluster) []SnapshotCl
 func (r *StorageClusterReconciler) createSnapshotClasses(vsccs []SnapshotClassConfiguration) error {
 
 	for _, vscc := range vsccs {
-		if vscc.reconcileStrategy == ReconcileStrategyIgnore || vscc.disable {
+		if vscc.reconcileStrategy == ReconcileStrategyIgnore {
```

Comment:
We also need similar change in the volumegroupsnapshotclass as well.

Reply:
@Madhu-1 the disable field in volumegroupsnapshotclass uses the Spec.AllowRemoteStorageConsumers field value.
https://github.com/red-hat-storage/ocs-operator/blob/f8864f74ba4a51da54702683c53ebd12cd599252/controllers/storagecluster/volumegroupsnapshotterclasses.go#L59

Reply:
If you removed the disable flag for `disable:           instance.Spec.ManagedResources.CephBlockPools.DisableSnapshotClass || instance.Spec.AllowRemoteStorageConsumers,` any specific reason to remove above one and keep this one check only for volumegroupsnaphotclass?

Reply:
Updated now